YourToDo/templates/static/about.html

```diff
@@ -35,23 +35,29 @@
 
-                        <p>Quisque justo odio, consectetur eget efficitur vel, euismod eu erat. Proin porttitor, mi eget tincidunt accumsan, lorem justo aliquet , efficitur finibus velit. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Nunc consectetur ipsum id venenatis condimentum. Mauris magna diam, dictum a neque ac, semper dignissim magna. Praesent mauris tellus, sagittis at velit vitae, faucibus posuere dolor.</p>
+                        <p>Ben recently completed an internship with Lexmark Enterprise Software (formerly known as Perceptive Software) this summer in the Kansas City area. Currently, he is devoted to being a full-time student at KU this fall semester of 2015.</p>
+
+                        <p>YourTodo was conceived by Ben and is now in the process of becoming an actual product through the help of colleagues Jay Offerdahl, Chad Papineau, and Wilson Hack.</p>
                     </div>                    
                     
                     <div class="col-lg-12 about-bio-sep">
                         <img class="about-profile-pic img-circle" src="{% static 'media/images/chad-profile.jpg' %}" alt="Chad Papineau's Picture" />
-                        <h3>Chad Papineau, Software Engineer 2B</h3>
-                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ullamcorper accumsan quam, sit amet ultrices enim porttitor ac. Phasellus nec convallis mi. Sed sit amet ante dui. Sed fringilla augue non lacinia vehicula. Quisque in ultricies magna. In id diam dui. Donec tempus elit et pellentesque faucibus. Pellentesque interdum risus libero, sed condimentum sapien malesuada sed. Donec viverra sapien nibh, eu dapibus sapien rutrum in. Duis porttitor mauris elit, vel viverra justo iaculis sit amet. In vel ipsum et purus pretium imperdiet. Donec et blandit est. Etiam non fermentum quam, id aliquam metus. Curabitur gravida volutpat finibus. Quisque feugiat erat in condimentum pharetra.</p>
+                        <h3>Chad Papineau, Full Stack Developer &amp; Business Solutions</h3>
+                        <p>Chad's a junior at the University of Kansas where I am seeking a B.S. in Computer Science and a Minor in Business.</p>
+
+                        <p>This past summer Chad internerd with H&amp;R Block's .com and myBlock teams. During my internship I worked with the development and modernization of their W-4 Calculator to create it as a single page web application utilizing the AngularJS framework.</p>
+
+                        <p>Outside of school, he is the Vice President of the university sponsored cyber security and competitive hacking team, A.K.A. "The Jayhackers". Along with the hacking team, I am also an active member of the KU Information Security Club.</p>
 
```

Comment:
Chad's about description differs using both first person and third person language. We need to be consistent about this as yours, Wilson's and my bios are all written in third person.